app/controllers/secondary_content_links_controller.rb

```diff
@@ -40,10 +40,6 @@ def secondary_content_link
     @secondary_content_link ||= step_by_step_page.secondary_content_links.find(params[:id])
   end
 
-  def update_downstream
-    StepByStepDraftUpdateWorker.perform_async(step_by_step_page.id, current_user.name)
```

Comment:
Instead of making a new class which just does one thing (a bit of a smell), I'd suggest exposing a new method we can call on `StepByStepDraftUpdateWorker` and just update the definition inside `update_downstream`.

Something like:

```ruby
def update_downstream
  StepByStepDraftUpdateWorker.
    update_status_sync(step_by_step_page).
    perform_async(step_by_step_page.id, current_user.name)
end
```

Would mean the only code that needs changing in each file is the `update_downstream` method.

Reply:
Fair! 👍